content/en/docs/setup/independent/high-availability.md

```diff
@@ -447,28 +447,29 @@ Only follow this step if your etcd is hosted on dedicated nodes (**Option 1**).
 ## Run kubeadm init on master0 {#kubeadm-init-master0}
 
 1. In order for kubeadm to run, you first need to write a configuration file:
-     ```bash
-     cat >config.yaml <<EOF
-     apiVersion: kubeadm.k8s.io/v1alpha1
-     kind: MasterConfiguration
-     api:
-      advertiseAddress: <private-ip>
-     etcd:
-      endpoints:
-      - https://<etcd0-ip-address>:2379
-      - https://<etcd1-ip-address>:2379
-      - https://<etcd2-ip-address>:2379
-      caFile: /etc/kubernetes/pki/etcd/ca.pem
-      certFile: /etc/kubernetes/pki/etcd/client.pem
-      keyFile: /etc/kubernetes/pki/etcd/client-key.pem
-     networking:
-      podSubnet: <podCIDR>
-     apiServerCertSANs:
-     - <load-balancer-ip>
-     apiServerExtraArgs:
-      apiserver-count: "3"
-     EOF
-     ```
+
+    {{< highlight bash >}}
```

Comment:
i wonder if this can be fixed with some sort of a different trick.
we kind of have consistency with using "```bash" everywhere, yet it appears that something is breaking it here. 

/assign @Bradamant3 
/assign @mistyhacks